package.json

```diff
@@ -24,5 +24,6 @@
   },
   "scripts": {
     "test": "./node_modules/.bin/karma start --browsers ChromeHeadless --single-run"
-  }
+  },
+  "version": "11.4.0"
```

Comment:
Sorry if I'm picking on this, but was this line here before? It seems like it may lead to some confusion when the package lock is updated.

Reply:
sorry - I thought I reverted this... This is added by npm, most likely because I'm using an updated version of it. I'll revert it.